### ISSUE TITLE
Admin Interface: Show ID in database of these entries

### ISSUE THREAD

**Issue by Gateway85** [37coins/37coins-web#41]
Should be possible to see ID of found entries... (new column) 
this holds for all three tables

**Comment by mahmed0715** [comment]
Account ID already shown : issue #52 ...

**Comment by Gateway85** [comment]
Do we do the same for gateways and markets? If not we should add the ID to them (markets should also still show the name)

**Comment by mahmed0715** [comment]
Yes, accounts, gateways and markets now has a column ID, market name is also showing.
